lib/paperclip/storage/google_drive.rb

```diff
@@ -156,7 +156,7 @@ def public_url_for title
       def search_for_title(title)
         parameters = {
                 'folderId' => find_public_folder,
-                'q' => "title contains '#{title}'", # full_title
```

Comment:
This is doable, though it it worth noting that the "contains" method doesn't work perfectly - for instance, files with long names won't return a URL.

What are the benefits you see in having it loose?